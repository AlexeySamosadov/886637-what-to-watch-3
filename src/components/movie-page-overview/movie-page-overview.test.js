import React from "react";
import renderer from "react-test-renderer";
import MoviePageOverview from "./movie-page-overview.jsx";

const ratingCount = 323;
const description = `Here Some Description`;
const directors = `Alex Smitch`;
const actors = [`Toni Hawk`, `Arnold Vicci`, `Van gog`, `Charli Chaplin`, `German French`, `Italian English`];
const ratingLevel = `mustSee`;


it(`This is Unit Test for MoviePageDetails component`, ()=>{
  const tree = renderer
    .create(
        <MoviePageOverview
          ratingLevel={ratingLevel}
          description={description}
          directors={directors}
          actors={actors}
          ratingCount={ratingCount}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
