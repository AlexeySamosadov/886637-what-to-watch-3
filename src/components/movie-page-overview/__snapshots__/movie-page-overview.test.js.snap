// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is Unit Test for MoviePageDetails component 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8,9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        mustSee
      </span>
      <span
        className="movie-rating__count"
      >
        323
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      dsfsdfdsfsfsdfsdfdsf
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Alex Smitch
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Toni Hawk, Arnold Vicci, Van gog, Charli Chaplin, German French, Italian English
      </strong>
    </p>
  </div>,
]
`;
